en/SUMMARY.md

```diff
@@ -19,6 +19,7 @@
       - [Mission Mode (MC)](flight_modes_mc/mission.md)
       - [Return Mode (MC)](flight_modes_mc/return.md)
       - [Offboard Mode (MC)](flight_modes_mc/offboard.md)
+    - [Collision Prevention](computer_vision/collision_prevention.md)
```

Comment:
@Claudio-Chies Moved this under MC features because unlike the other computer vision features, this works with a locally attached camera. I think worth highlighting (is still linked from computer vision).

Does this also work with VTOL in MC mode?